static/robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: /
+Allow: /account*
```

Comment:
I'm not sure if we want or need robots traffic on account pages.
In this case I guess we should `Disallow: /account`